templates/generate_client_api.sh

```diff
@@ -37,4 +37,4 @@ generate_files() {
 generate_files https://raw.githubusercontent.com/symphonyoss/symphony-api-spec/master/agent/agent-api-public.yaml agent
 generate_files https://raw.githubusercontent.com/symphonyoss/symphony-api-spec/master/authenticator/authenticator-api-public.yaml auth
 generate_files https://raw.githubusercontent.com/symphonyoss/symphony-api-spec/master/login/login-api-public.yaml login
-generate_files https://raw.githubusercontent.com/symphonyoss/symphony-api-spec/master/pod/pod-api-public-deprecated.yaml pod
+generate_files https://raw.githubusercontent.com/symphonyoss/symphony-api-spec/master/pod/pod-api-public.yaml pod
```

Comment:
@symphony-youness Why did you changed it ? If I remember well, some endpoints were missing in the non deprecated pod api

Reply:
I thought it was a mistake 🤔

Reply:
I tried to recompile the java bdk with 20.12 specs and pod-api-public.yaml and it seems fine. So it is fine to use the pod-api-public.yaml